src/backend/projects/project_routes.py

```diff
@@ -71,8 +79,10 @@ async def read_project(project_id: int, db: Session = Depends(database.get_db)):
 @router.post("/delete/{project_id}")
 async def delete_project(project_id: int, db: Session = Depends(database.get_db)):
     """Delete a project from ODK Central and the local database"""
-    # FIXME: should check for errors
+    # FIXME: should check for error
     odkproject = central_crud.delete_odk_project(project_id)
+    # if not odkproject:
```

Comment:
Nice to see some error handling here; very much appreciated. It's too easy to not bother dealing with such cases (an attempt to manage an asset that doesn't exist on a remote machine), and we've definitely been guilty of putting off this kind of housekeeping on this project so far. Kudos.

Reply:
There's not a whole lot of error handling in this PR, that'll be the next pass. This was just getting it to work.

Reply:
Nevertheless it's encouraging to see some of it happening already.